English Curriculum: Block 17 (#59564)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-about-speculation-and-requests/67f666c9821e1e1ed24333b2.md

---
id: 67f666c9821e1e1ed24333b2
title: Task 148
challengeType: 19
dashedName: task-148
---

<!-- (audio) Brian: Let me know if you have any other suggestions. -->

# --instructons--

Listen to the audio and answer the question below.

# --questions--

## --text--

What is Brian asking Alice to do?

## --answers--

He is telling Alice to stop making suggestions.

### --feedback--

Brian is asking for more suggestions, not stopping them.

---

He wants Alice to share any other suggestions she has.

---

He is asking Alice to lead the training session.

### --feedback--

Brian does not ask Alice to lead the training.

---

He wants Alice to approve his plan without changes.

### --feedback--

Brian is open to suggestions and does not ask for automatic approval.

## --video-solution--

2

# --explanation--

`Let me know` is a polite way to ask for information or feedback. For example:

> `Let me know your thoughts on the new design.` - Someone is asking you to share your opinion about the design.

Reviewer: ```suggestion
`Let me know your thoughts on the new design.` - You are asking someone to share opinions about the design.
```
